src/sentry/api/endpoints/organization_teams.py

```diff
@@ -46,16 +45,11 @@ class OrganizationTeamsPermission(OrganizationPermission):
 class TeamPostSerializer(serializers.Serializer):
     name = serializers.CharField(max_length=64, required=False, allow_null=True, allow_blank=True)
     slug = serializers.RegexField(
-        r"^[a-z0-9_\-]+$",
+        DEFAULT_SLUG_PATTERN,
```

Comment:
If you go with [this option](https://github.com/getsentry/sentry/pull/55122#discussion_r1300521992) then you don't have to edit this.